usr/src/boot/forth/brand-smartos.4th

```diff
@@ -21,17 +22,17 @@
 	1+ \ increase y for next time we're called
 ;
 
-: brand ( x y -- ) \ "joyent" [wide] logo in B/W (7 rows x 42 columns)
+: brand ( x y -- ) \ "triton" [wide] logo in B/W (7 rows x 42 columns)
 
-	0 1 1 0 0 s" /boot/joyent.png" fb-putimage if 2drop exit then
+	0 1 1 0 0 s" /boot/triton-logo.png" fb-putimage if 2drop exit then
 
-	s"       #                                   " brand+
-	s"       #  ####  #   # ###### #    # #####  " brand+
-	s"       # #    #  # #  #      ##   #   #    " brand+
-	s"       # #    #   #   #####  # #  #   #    " brand+
-	s" #     # #    #   #   #      #  # #   #    " brand+
-	s" #     # #    #   #   #      #   ##   #    " brand+
-	s"  #####   ####    #   ###### #    #   #  TM" brand+
+	s"                                           " brand+
+	s" ####### #####  # #######   ###   #    #   " brand+
+	s"    #    #      #    #     #   #  ##   #   " brand+
+	s"    #    #      #    #    #     # # #  #   " brand+
+	s"    #    #      #    #    #     # #  # #   " brand+
+	s"    #    #      #    #     #   #  #   ##   " brand+
+	s"    #    #      #    #      ###   #    # TM" brand+
```

Comment:
I stylized the text logo a bit in a way that I like better:

For posterity, this is a slight modification to the clr6x8 figlet font.

```
        s"                                           " brand+
        s" #####  ####   #  #####   ###   #   #      " brand+
        s"   #    #   #  #    #    #   #  ##  #      " brand+
        s"   #    ####   #    #    #   #  # # #      " brand+
        s"   #    # #    #    #    #   #  #  ##      " brand+
        s"   #    #  #   #    #    #   #  #  ##      " brand+
        s"   #    #   #  #    #     ###   #   # TM   " brand+
```

Reply:
Will fix.